vitest.config.ts

```diff
@@ -1,9 +1,11 @@
 import { defineConfig } from "vitest/config";
 
+process.env.NEXT_PUBLIC_UNIT_TESTS = 1;
```

Comment:
This is to convey to _baseEmail that you need to collect emails in memory and not send them so that integration tests can verify that. I am thinking of renaming it to NEXT_PUBLIC_INTEGRATION_TESTS

Reply:
Reused existing INTEGRATION_TESTS_MODE env variable here.